### DISCUSSION TITLE
[Feature] Add download progress

### DISCUSSION THREAD

**Discussion by RWDai** [immich-app/immich#10100] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

#7379  Similar should add download progress

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by Fricsion** [comment]
In my humble opinion, this is a much needed feature. 

On my iPhone, I have no way to see how long more it takes to download a 800MB video. (And I'm not patient enough to wait for that not knowing how long it will take)

**Comment by hermesespinola** [comment]
On iOS, if I exit the app after clicking download, it never completes, it'd be cool if there was a way to put downloads as a background process?

**Comment by Acidiphilium** [comment]
I agree it would be a nice feature and same for the downloads launched as a background process.

**Comment by bo0tzz** [comment]
Will be closed by  #12973
